src/modules/submitGame.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+const handleSubmit = async (event) => {
+  event.preventDefault();
+  const nameInput = document.getElementById('name');
+  const scoreInput = document.getElementById('score');
+  const name = nameInput.value;
+  const score = scoreInput.value;
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/wUcdYlqfnVytiFkqhyG8/scores/';
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+
+      body: JSON.stringify({ user: name, score: score }),
+
+    }).then((data) => data.json());
+    console.log(response);
+  } catch (error) {
+    throw error;
+  }
+};
```

Comment:
- [ ] Nice work so far 👍🏿 as per project requirement when the user enters their input data and hits the submit button it's supposed to make a post. Request to the API but currently it does not I kindly suggest you fix that bug :wink